README.md

```diff
@@ -142,6 +142,29 @@ Response onRequest(RequestContext context) {
 }
 ```
 
+We can also return any Dart object in the `body` of the  `Response.json` constructor as long as it has a `toJson` and `fromJson` function (for example, using [json_serializable](https://pub.dev/packages/json_serializable)):
```

Comment:
```suggestion
We can also return any Dart object in the `body` of the  `Response.json` constructor and it will be serialized correctly as long as it has a `toJson` method that returns a `Map<String, dynamic>`. Check out [json_serializable](https://pub.dev/packages/json_serializable) to automate the `toJson` generation.
```